packages/storage/amplify_storage_s3_dart/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.1.5
+
+### Fixes
+- fix(storage): incorrect transferred bytes emitted from upload task
+
+### Features
+- feat(storage): allow configuring transfer acceleration
+
```

Comment:
Do these two items list in the CHNAGELOG of amplify_flutter package as well?

Reply:
This isn't currently supported in aft, so we'll need to come up with a mechanism for it